FRCore/FRCore/Log/Log.swift

```diff
@@ -164,6 +164,14 @@ public class Log: NSObject {
         Log.logLevel = logLevel
     }
     
+    /// Sets Custom Logger
+    ///
+    /// - Parameter logger: Set logger will reset the logger to Custom Logger, Default Logger will be inactive
+    @objc
+    public static func setCustomLogger(_ logger: FRLogger) {
+        self.loggers = [logger]
```

Comment:
Overall looks good, but I have a more generic question regarding this Logger code. Why is the Loggers an Array if we only use one? As far as I see there is not way to have 2 running the same time in our current implementation.

Reply:
@vahancouver @jeyanthanperiyasamy @rodrigoareis If the above observation is correct we could change that and simplify the code of all of those classes